src/wallet/rpcwallet.cpp

```diff
@@ -847,7 +864,23 @@ UniValue supportclaim(const UniValue& params, bool fHelp)
         );
 
     string sName = params[0].get_str();
+
+    if (sName.empty())
+        throw JSONRPCError(RPC_WALLET_ERROR, "Claim name should not be empty");
+
     string sClaimId = params[1].get_str();
+
+    if (!((sClaimId[0] == 0 && tolower(sClaimId[1]) == 'x' && sClaimId.size() == 162) || sClaimId.size() == 160))
+        throw JSONRPCError(RPC_WALLET_ERROR, "Claim id should be exact 160 symbols");
+
+    size_t i = 0;
+    if (sClaimId[0] == 0 && tolower(sClaimId[1]) == 'x')
+        i = 2;
+
+    for (; i < sClaimId.size(); ++i)
+        if (!isxdigit(sClaimId[i]))
+            throw JSONRPCError(RPC_WALLET_ERROR, "Claim id should contains only hexdecimal numbers");
+
```

Comment:
There is actually a function created to check claimId here: 
https://github.com/lbryio/lbrycrd/blob/master/src/rpc/claimtrie.cpp#L10